repos/spack_repo/builtin/packages/rivet/package.py

```diff
@@ -154,6 +156,9 @@ def configure_args(self):
             args += ["--with-hepmc3=" + self.spec["hepmc3"].prefix]
             args += ["--with-hepmc3-libpath=" + self.spec["hepmc3"].libs.directories[0]]
 
+        if not self.spec.satisfies("+analyses"):
+            args += ["--with-analyses=no"]
```

Comment:
Does this option support anything other than yes/no? Should we approach this like `openloops` and allow for users to specify subsets of analyses?

Reply:
And `psblas` accepts free-form `ccopt`, `cxxopt`, etc.

Reply:
Alternative is to provide a few pre-selected options, e.g. `[exp for plugin${exp} in analyses]`.